src/util/floatingpoint_literal_symfpu_traits.cpp

```diff
@@ -110,35 +110,35 @@ template <bool isSigned>
 wrappedBitVector<isSigned> wrappedBitVector<isSigned>::operator|(
     const wrappedBitVector<isSigned>& op) const
 {
-  return BitVector::operator|(op);
+  return static_cast<BitVector>(*this) | static_cast<const BitVector&>(op);
```

Comment:
It would be a bit better to change all the `static_cast<BitVector>` to `static_cast<BitVector&>`